ansible-runner/context/app/project/roles/mirror_appliance/tasks/secret-vars.yaml

```diff
@@ -0,0 +1,6 @@
+- name: Generate secrets for Quay config.yaml
```

Comment:
I'm not very familiar with the `community.general.random_string`. A quick google search yields its powered by the https://docs.python.org/3/library/random.html#random.SystemRandom lib. It looks like it will use a unique seed on each invocation unless specified specific seed (for testing purposes). 

However, it does mention limitations of not being available on some OS, though it doesn't call out any. Do we know what the limitations of this role and underlying libraries are?

Reply:
Yeah, I'm not sure exactly why that doc mentions it not being available - my guess is maybe a *BSD? Looks like even windows can deal with `os.urandom` properly.

Given that it is highly unlikely this tool will be installed on a non RHEL OS, I don't think we'll see an issue. The primary reason I went this path was that `community.general` was already installed, and I didn't want to add any more imports/complexity than necessary to accomplish the goal.